config/default_records/request_types/020_limber_high_throughput_bge.wip.yml

```diff
@@ -0,0 +1,37 @@
+---
+# Y24-366
+limber_bge_pcr_free:
+  asset_type: Well
+  order: 1
+  request_class_name: IlluminaHtp::Requests::StdLibraryRequest
+  billable: true
+  product_line_name: Illumina-HTP
+  name: Limber BGE PCR Free
+  acceptable_purposes:
+    - PF Cherrypicked
+  library_types:
+    - BGE
+# For moving PCR Free plates to ISC plates
+# e.g., BGE Lib XP2 -> BGE LIB PCR
+limber_bge_transition:
+  asset_type: Well
+  order: 1
+  billable: true
+  product_line_name: Illumina-HTP
+  name: Limber BGE Transition
+  request_class_name: IlluminaHtp::Requests::StdLibraryRequest
+  acceptable_purposes:
+    - LB Cherrypick
+  library_types:
+    - BGE
+limber_bge_isc:
+  asset_type: Well
+  order: 1
+  billable: true
+  product_line_name: Illumina-HTP
+  name: Limber BGE ISC
+  request_class_name: Pulldown::Requests::IscLibraryRequest
+  acceptable_purposes:
+    - LB Cherrypick
```

Comment:
Thinking about this more this should be technically more of a ReISCLibraryRequest because it joins the ISC pipeline half way down. Apologies the story description was wrong. The acceptable purpose here will also be a `BGE Lib PCR XP`